lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -27,30 +32,10 @@ export abstract class PerformanceClient implements IPerformanceClient {
 
     /**
      * Multiple events with the same correlation id.
-     * Double keyed by correlation id and event id.
      * @protected
-     * @type {Map<string, Map<string, PerformanceEvent>>}
-     */
-    protected eventsByCorrelationId: Map<string, Map<string, PerformanceEvent>>;
-
-    /**
-     * Fields to be emitted which are scoped to the top level request and whose value will not change in submeasurements
-     * For example: App name, version, etc.
+     * @type {Map<string, PerformanceEvent>}
      */
-    protected staticFieldsByCorrelationId: Map<string, StaticFields>;
```

Comment:
Removing these is a breaking change. This should be a major bump of common

Reply:
I would rather disagree because:
1. It's does not change API signature
2. We don't have any classes inheriting base `PerformanceClient` class that access `staticFieldsByCorrelationId` directly.

Reply:
`PerformanceClient` is exported from common therefore removing anything public or protected is breaking. It's not a big deal though, we just need to mark this as a major change in the msal-common changefile. Please do make sure anything that is not used outside of this class is marked `private` to avoid this churn in the future though.

Reply:
Sounds good. Changed to major version bump.